packages/apps-v5/src/commands/ps/index.js

```diff
@@ -68,25 +68,13 @@ async function printAccountQuota(context, heroku, app, account) {
   let appHours = Math.floor(appQuotaUsed / 60)
   let appMinutes = Math.floor(appQuotaUsed % 60)
 
-  if (app.process_tier === 'eco') {
-    cli.log(`Eco dyno hours quota remaining this month: ${hours}h ${minutes}m (${percentage}%)`)
-    cli.log(`Eco dyno usage for this app: ${appHours}h ${appMinutes}m (${appPercentage}%)`)
-    cli.log('For more information on Eco dyno hours, see:')
-    cli.log('https://devcenter.heroku.com/articles/eco-dyno-hours')
-    cli.log()
-  }
-
-  if (app.process_tier === 'free') {
```

Comment:
I know that on Nov 28th that free apps will still be hanging around because the script is supposed to take around a month to fully convert everyones apps. 


I'm wondering if this should stay until that is done, and we have ANOTHER branch for post-conversion updates to clean up any remaining free/hobby stuff? We can safely remove anything that involves **creating** free/hobby dynos, but any code that is needed for existing ones to continue working I am thinking might need to stay... @tevanoff @eablack

Reply:
Thanks for the reminder! I'll add that logic back.